src/lib/mollie/api/resource/base.coffee

```diff
@@ -51,31 +51,39 @@ module.exports = class Base
 				if (body.error)
 					return callback(body);
 				callback(this.copy(body, new this.constructor.object));
-		);
+		).on("error", (error) => 
```

Comment:
Valid point. However the callback parameter of the methods (get, put, post, etc.) in base class are not exposed to the callrest method in the client. Any suggestion how to work around this? Perhaps, provide an additional argument to callrest with an error callback funciton?